docs/dashboards/rule-monitoring-dashboard.asciidoc

```diff
@@ -0,0 +1,63 @@
+[[rule-monitoring-dashboard]]
+= Detection rule monitoring dashboard
+
+:frontmatter-description: Visualize your detection rules' performance.
+:frontmatter-tags-products: [security]
+:frontmatter-tags-content-type: [how-to]
+:frontmatter-tags-user-goals: [visualize, monitor]
+
+The Detection rule monitoring dashboard provides visualizations to help you monitor the overall health and performance of {elastic-sec}'s detection rules. Consult this dashboard for a high-level view of whether your rules are running successfully and how long they're taking to run, search data, and create alerts.
+
+[role="screenshot"]
+image::images/rule-monitoring-overview.png[Overview of Detection rule monitoring dashboard]
+
+.Requirements
+[sidebar]
+--
+To access this dashboard and its data, you must have:
+
+* At least `Read` {kibana-ref}/kibana-role-management.html#adding_kibana_privileges[{kib} privileges] for both the `Analytics > Dashboard` and `Security > Security` {kib} features.
```

Comment:
Small nit/question, but why are we wrapping `Analytics > Dashboard` and `Security > Security`  in code text instead of bold?

Reply:
Whoops, that's a mistake, I didn't look at the preview on this. I'll change to bold.